src/transparency/h1.rs

```diff
@@ -62,10 +62,10 @@ fn set_authority(uri: &mut http::Uri, auth: Authority) {
 
     // If this was an origin-form target (path only),
     // then we can't *only* set the authority, as that's
-    // an illegal target (such as `conduit.io/docs`).
+    // an illegal target (such as `linkerd.io/docs`).
```

Comment:
Let's use `example.com` or `example.org` for example domains where the actual domain name doesn't matter (here and elsewhere throughout this PR). If we'd done that before we'd not need to do this now, and also using `example.com`/`example.org` makes it clearer to readers that the actual domain doesn't matter.